include/inexor/vulkan-renderer/wrapper/descriptor_builder.hpp

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <vulkan/vulkan_core.h>
+
+#include <cassert>
+#include <string>
+#include <vector>
+
+namespace inexor::vulkan_renderer::wrapper {
+
+class Device;
+class ResourceDescriptor;
+
+class DescriptorBuilder {
+    const Device &m_device;
+    const std::uint32_t m_swapchain_image_count{};
```

Comment:
```suggestion
    const std::uint32_t m_swapchain_image_count;
```
No zero-initialisation needed since it's always set by constructor.

Reply:
Done.